$MirOS: ports/devel/p5-SDL/patches/patch-Makefile_PL,v 1.1 2005/06/24 17:24:07 tg Exp $
--- Makefile.PL.orig	Tue Jun 11 02:12:48 2002
+++ Makefile.PL	Fri Nov  3 23:09:46 2006
@@ -8,6 +8,8 @@
 	linux 	=> "Makefile.linux",
 	win 	=> "Makefile.win32",
 	freebsd => "Makefile.freebsd",
+	mirbsd	=> "Makefile.netbsd",
+	openbsd	=> "Makefile.netbsd",
 );
 
 #
